### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by kamena1994**
## Task: `html-task`
- `added e-commence`

### CODE REVIEW

**Comment by ToluAkin** [comment]
You are actually required to added screenshots of the e-commerce page on the web

File: submissions/benjaminKamena/e-commerce/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/benjaminKamena/e-commerce/index.html:1]: Well done!

**Comment by kamena1994** [comment]
let me do that

**Comment by kamena1994** [comment]
I have made the changes

**Comment by kamena1994** [comment]
I have changed the naming of the image

**Comment by kamena1994** [comment]
i feel am not understanding the question

**Comment by ToluAkin** [comment]
> i feel am not understanding the question

In the instructions for this task, you were asked to use only 2 product images but you have 6 images. Remove the extra images and delete the unused ones. Check [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7) for the instruction

**Pull request closed without merging**